README.md

```diff
@@ -214,6 +217,27 @@ And then installing again:
 ./install --force
 ```
 
+#### Missing symbols & ligatures in Vim (Linux)
+
+If symbols appear corrupted in Vim, first ensure that your terminal emulator supports UTF-8. Then, check to see if you've properly configured your system-wide UTF-8 support.
+
+Issue this test command:
+
+```bash
+export LC_ALL=en_US.UTF-8
+```
+
+Then run `vim`. Powerline symbols & statusline ligatures should be working.
```

Comment:
Cool, thanks for clarifying. "[Ligatures](https://en.wikipedia.org/wiki/Typographic_ligature)" isn't the right word here (see [this article](https://visualstudioextensions.vlasovstudio.com/2017/05/01/improve-code-readability-with-programming-ligatures-in-fira-code-font/) for some example screenshots). I'll just update this to say "Unicode characters" or something like that.